Styles.css

```diff
@@ -0,0 +1,42 @@
+body {
+  text-align: center;
+}
+
+.sub-div {
+  max-width: 500px;
+}
+
+.div-main {
+  display: flex;
+  justify-content: center;
+}
+
+main {
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
+  background-color: #eee;
+}
+
+#image {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: auto;
+}
+
+#img-caption {
+  margin: 15px 0 5px 0;
+}
+
+ul {
+  text-align: left;
+}
+
+blockquote {
+  text-align: left;
+}
+
+li {
+  margin: 16px 0;
+}
```

Comment:
I think you should add a `box-sizing: border-box. This will help manage the extra padding and borders, thereby enabling you to keep track of element width. 👍 Just like this

`* {
  box-sizing: border-box;
}`

Reply:
Giving an insight of what you are requesting is awesome 👌 _Don't forget to use please/kindly to avoid your comments sounding like an order rather than a suggestion 😁_